examples/nightlight/nightlight.py

```diff
@@ -0,0 +1,98 @@
+#!/bin/env python
+
+import time
+from ltr559 import setup, update_sensor, get_lux, get_proximity
```

Comment:
I can't say I'm the biggest fan of importing methods out of their namespace into global since it makes the call to `setup` further down the code ambiguous.

Reply:
That's from your example in the LTR-559 library, dude! But I agree... :-D